README.md

```diff
@@ -176,7 +176,8 @@ err = env.DeclareStream(streamName,
 		SetMaxLengthBytes(stream.ByteCapacity{}.GB(2)))
 ```
 
-Note: The function `DeclareStream` returns `stream.StreamAlreadyExists` if a stream is already defined.
+The function `DeclareStream` doesn't return errors if a stream is already defined with the same parameters.
```

Comment:
Maybe note that it returns the precondition failed when it doesn't have the same parameters